File: FrontendTemplateProject-main/src/app/models/UserStoryModel/functionalityCreationDto.ts
import { Guid } from 'guid-typescript';

export class FunctionalityCreationDto {
    
    textFunctionality!: string;
    userStoryRootId!: Guid;

    constructor(
        textFunctionality: string,
        userStoryRootId: Guid
    ) {
        this.textFunctionality = textFunctionality;
        this.userStoryRootId = userStoryRootId;
    }
    
}

File: UserStory/UserStory/Models/ModelFunctionality/FunctionalityCreationDTO.cs
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using UserStory.Models.ModelUserStory;

namespace UserStory.Models.ModelFunctionality
{
    public class FunctionalityCreationDTO
    {
        //public Guid FunctionalityId { get; set; }
        public string TextFunctionality { get; set; }
        public Guid UserStoryRootId { get; set; }
        public Guid? SprintId { get; set; }
    }
}
